openwisp_controller/config/tests/utils.py

```diff
@@ -90,6 +94,7 @@ class CreateVpnMixin(object):
     }
 
     def _create_vpn(self, ca_options={}, **kwargs):
+        setattr(subprocess, 'check_output', lambda *args, **kargs: self._dh.encode())
```

Comment:
@nemesisdesign 

Sorry, I still didn't understand what you meant by : `I think we should mock subprocess.check_output in Vpn.dhparam` in the issue.

I assume you meant during testing we should mock the `subprocess.check_output` function & I made this change, but only one test was affected by it, so I am not sure if it's really what you meant or if it's worth to mock it!